.circleci/config.yml

```
@@ -49,7 +49,12 @@ jobs:
 
   e2eTestChartInstall:
     environment:
-      TEST_DIR: "integration/test/chart"
+      TEST_DIR: "integration/test/app"
```

```suggestion
      TEST_DIR: "integration/test/app/basic"
```

Good the test dir is changed to app but the test needs to be in its own directory.